include/cantera/zeroD/flowControllers.h

```diff
@@ -84,26 +94,57 @@ class Valve : public FlowDevice
 public:
     Valve();
 
+<<<<<<< HEAD
     virtual std::string typeStr() const {
         return "Valve";
     }
 
     virtual bool ready() {
         return FlowDevice::ready() && (m_coeffs.size() == 1 || m_func);
+=======
+    //! Set the proportionality constant between pressure drop and mass flow
+    //! rate
+    /*!
+     * *c* has units of kg/s/Pa. The mass flow rate is computed as:
+     * \f[\dot{m} = c \Delta P \f]
+     */
+    //! @deprecated To be removed after Cantera 2.5.
+    void setPressureCoeff(double c) {
+        warn_deprecated("Valve::setParameters()",
+                        "To be removed after Cantera 2.5. "
+                        "Use Valve::setValveCoeff instead.");
+        m_coeff = c;
+>>>>>>> bfab8a070... [Reactor] clarified FlowDevice interface
     }
 
     //! Set the proportionality constant between pressure drop and mass flow
     //! rate
     /*!
      * *c* has units of kg/s/Pa. The mass flow rate is computed as:
+     * \f[\dot{m} = c l(t) f(\Delta P) \f]
+     * where *l* and *f* are functions of time and pressure drop that are set
+     * by `setValveLift` and `setFunction`, respectively. If no functions are
```

Comment:
Isn't part of the confusion here that it's unclear which function `setFunction` should set? Shouldn't the method to set f(dP) be called something like `setPressureFunction`?

Reply:
`setPressureFunction` it is.